src/site/docs/synonyms/assets/es6-samples.xml

```diff
@@ -0,0 +1,1038 @@
+<?xml version="1.0"?>
+<synonyms>
+<theme id="theme-getting-started">
+  <title>Getting started</title>
+  <synonym id="syn-code-embedding">
+    <title>Code embedding</title>
+    <code language="javascript"><![CDATA[
```

Comment:
I think simply through https://github.com/google/code-prettify but I'm pretty sure we're running an old version of that.